config/database.yml

```diff
@@ -19,9 +19,9 @@ default: &default
   encoding: unicode
   database: postgres
   pool: 5
-  username: postgres
-  password: mysecretpassword
-  host: db
+  username: <%= ENV['RUNNING_LOCALLY'] ? ENV['DATABASE_USERNAME'] : 'postgres' %>
```

Comment:
I would try to change approach here. As you can see under `production` section, we already have a `DATABASE_URL` environmental variable in use there.

I would suggest to change this file that - for `development` and `test` - we can also reuse this variable, so if it's defined, it's simply used (the URL can contain all 3 things, username, password, and host), and overrides the default values; however, if it's not defined, these 3 values should fallback to the default values (`postgres`, `mysecretpassword`, `db`).